$OpenBSD: patch-tools_clang_lib_Driver_ToolChains_cpp,v 1.3 2015/02/11 00:29:05 brad Exp $

r225958
Use the integrated assembler by default on 32-bit PowerPC and SPARC.

--- tools/clang/lib/Driver/ToolChains.cpp.orig	Mon Feb  2 11:00:57 2015
+++ tools/clang/lib/Driver/ToolChains.cpp	Mon Feb  2 11:03:43 2015
@@ -2035,7 +2035,12 @@ bool Generic_GCC::IsIntegratedAssemblerDefault() const
          getTriple().getArch() == llvm::Triple::aarch64 ||
          getTriple().getArch() == llvm::Triple::aarch64_be ||
          getTriple().getArch() == llvm::Triple::arm ||
-         getTriple().getArch() == llvm::Triple::thumb;
+         getTriple().getArch() == llvm::Triple::thumb ||
+         getTriple().getArch() == llvm::Triple::ppc ||
+         getTriple().getArch() == llvm::Triple::ppc64 ||
+         getTriple().getArch() == llvm::Triple::ppc64le ||
+         getTriple().getArch() == llvm::Triple::sparc ||
+         getTriple().getArch() == llvm::Triple::sparcv9;
 }
 
 void Generic_ELF::addClangTargetOptions(const ArgList &DriverArgs,
